src/libraries/System.IO.FileSystem.AccessControl/README.md

```diff
@@ -0,0 +1,12 @@
+# System.IO.FileSystem.AccessControl
+This assembly provides support for controling access to and auditing security-related actions on securable objects.
+This assembly is only supported on Windows.
+
+Documentation can be found at https://learn.microsoft.com/dotnet/api/system.security.accesscontrol and https://learn.microsoft.com/dotnet/api/system.io.filesystemaclextensions.
+
+## Contribution Bar
+- [x] [We consider new features, new APIs and performance changes](../../libraries/README.md#primary-bar)
```

Comment:
I am not sure whether we consider new features for this library. It's Windows-only and afaik we have added everything that was needed to move from .NET Framework to Core.

```suggestion
- [x] [We only consider fixes to maintain or improve quality](../../libraries/README.md#primary-bar)
```